apps/codeforafrica/src/components/FooterMenu/index.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavList/NavListItem";
+
+function FooterMenu({ menu }) {
+  if (!menu?.length) {
+    return null;
+  }
+
+  return (
+    <NavList footer>
+      {menu.map((item) => (
+        <NavListItem
+          label={item.label}
+          key={item.label}
+          href={item.href}
```

Comment:
```suggestion
          {..item}
```
Since the prop names match, we can just spread them over the component.